cli/src/helper.rs

```diff
@@ -152,10 +127,10 @@ pub async fn generate_mnemonic(
         println_log_info!("Mnemonic has been written to '{file_path}'.");
     }
 
-    println_log_info!("IMPORTANT:");
-    println_log_info!("Store this mnemonic in a secure location!");
-    println_log_info!(
-        "It is the only way to recover your account if you ever forget your password and/or lose the stronghold file."
+    println!("IMPORTANT:");
```

Comment:
Why not println_log_info anymore?

Reply:
what's the purpose of this safety message in the log?

Reply:
I thought we just print everything (apart from the mnemonic) this way just so one can see what happened, but just checked and we mix both ways, so fine if we don't do it here